<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIChat</class>
 <widget class="QDialog" name="AIChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI 图像异常分析</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #F0F2F5;
    font-family: &quot;Microsoft YaHei UI&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E3A8A;</string>
     </property>
     <property name="text">
      <string>AI 图像异常分析</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="imageNameLabel">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2C3E50;</string>
     </property>
     <property name="text">
      <string>未选择图像</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="imageScrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>570</width>
        <height>148</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="chatTextBrowser">
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    border: 1px solid #E0E0E0;
    border-radius: 5px;
    background-color: #FFFFFF;
    padding: 8px;
    font-family: &quot;Microsoft YaHei UI&quot;;
    font-size: 10pt;
    color: #333333;
}</string>
     </property>
     <property name="placeholderText">
      <string>AI 分析结果将显示在这里...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="questionEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #D9D9D9;
    border-radius: 5px;
    background-color: white;
    padding: 5px 10px;
    font-family: &quot;Microsoft YaHei UI&quot;;
    font-size: 10pt;
}
QLineEdit:focus {
    border: 1px solid #1890FF;
    outline: none;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入你想询问的问题...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #9B59B6;
    color: white;
    border: none;
    border-radius: 5px;
    font-family: &quot;Microsoft YaHei UI&quot;;
    font-weight: bold;
    padding: 5px;
}
QPushButton:hover {
    background-color: #8E44AD;
}
QPushButton:pressed {
    background-color: #6C3483;
}</string>
       </property>
       <property name="text">
        <string>发送问题</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #E0E0E0;
    color: #333333;
    border: none;
    border-radius: 5px;
    font-family: &quot;Microsoft YaHei UI&quot;;
    font-weight: bold;
    padding: 5px;
}
QPushButton:hover {
    background-color: #D0D0D0;
}
QPushButton:pressed {
    background-color: #C0C0C0;
}</string>
       </property>
       <property name="text">
        <string>关闭对话</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 